### ISSUE TITLE
проблема с экшн кнопками у баллонов

### ISSUE THREAD

**Issue by simb11** [TauCetiStation/TauCetiClassic#12139]
#### Подробное описание проблемы
если положить подключенный к маске баллон на пол или в рюкзак, иконка экшн кнопки не обновится и будет выглядеть так, будто бы баллон подключён к маске
#### Что должно было произойти
экшн иконка обновляется когда баллон отключается не через кнопку или тгуименю
#### Что произошло на самом деле
не
#### Как повторить
взять любой баллон в руку, подключить его, бросить на пол, снова поднять и посмотреть на экшн иконку.
#### Дополнительная информация:
не знаю как это пофиксить, ведь эти кнопки одновременно привязаны и к предметам, и к мобам...

**Comment by ViDLiathan** [comment]
Верните иконку справа...

**Comment by Kiromaka** [comment]
это было до рефракта экшн кнопок ?

**Comment by pro100kotah** [comment]
> это было до рефракта экшн кнопок ?

Нет. Это случилось, когда отдельную иконку для баллона удалили и перенесли вверх. Вообще не одобряю этот ПР и не понимаю зачем из худа специальную иконку для баллона решили удалить. ПР ради ПРа.

**Comment by Kiromaka** [comment]
> > это было до рефракта экшн кнопок ?
> 
> Нет. Это случилось, когда отдельную иконку для баллона удалили и перенесли вверх. Вообще не одобряю этот ПР и не понимаю зачем из худа специальную иконку для баллона решили удалить. ПР ради ПРа. Который теперь багуется.

зарефрактили - значит используем). Рефракт и пры  к нему дохуя сломали ( экшн кнопки боргов и т.д)

**Comment by simb11** [comment]
> вообще не одобряю этот ПР и не понимаю зачем из худа специальную иконку для баллона решили удалить

может нужно читать описание пра и в особенности пункт "Почему и что этот ПР улучшит"? к нему был прикреплен ишшуй в котором написано с какой целью был сделан пр и для чего
